src/components/converter/Form.js

```diff
@@ -139,6 +141,17 @@ function useConvertInputs() {
 }
 
 function FormSection() {
+  const [selectedOption, setSelectedOption] = useState(0)
+  const [tokenBalance, setTokenBalance] = useState(bigNum(0))
+  const options = useMemo(() => ['ANT', 'DAI', 'ETH', 'USD'], [])
+  const selectedTokenBalance =
+    useTokenBalance(options[selectedOption]) || bigNum(0)
+  const ethBalance = useEthBalance()
+
+  useEffect(() => {
+    setTokenBalance(selectedTokenBalance)
```

Comment:
Should we set `tokenBalance` directly from `useSelectedTokenBalance` rather than doing it in two steps?